libs/live-config/src/providers/firebaseRemoteConfig/parser.ts

```diff
@@ -0,0 +1,27 @@
+import { Value } from "../firebaseRemoteConfig";
+import { ConfigInfo } from "../../LiveConfig";
+
+const objectParser = (value: unknown): object | undefined => {
+  const stringValue = (value as Value).asString();
+  return stringValue ? JSON.parse(stringValue) : undefined;
+};
+
+export function parser(value: unknown, type: ConfigInfo["type"]) {
```

Comment:
Why do we have to access it with ["type"] ?

Reply:
As we have 
```
type ConfigInfoShape<Type extends keyof ValidConfigTypes> = {
  type: Type;
  default: ValidConfigTypes[Type];
};

export type ConfigInfo =
  | ConfigInfoShape<"string">
  | ConfigInfoShape<"boolean">
  | ConfigInfoShape<"number">
  | ConfigInfoShape<"object">
  | ConfigInfoShape<"array">;

```

the ConfigInfo["type"] accepts "string", "boolean", "number", "array" and "object". Do you have any suggestion?

Reply:
Oops indeed it's fine